README.md

```diff
@@ -1,2 +1,223 @@
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="wahab_icon.jpeg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Capstone Project Module 1</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse Capstone Project Module 1 <a name="about-project"></a>
+
+This is My Capstone Project of Module 1 in Microverse which is A Dynamic and Responsive Website of A Non Profit Organization to Raise Awareness and Funds for The Flood Victims of 2022 in Pakistan.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### 1- Html
+### 2- CSS
+### 2- Javascript
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a>Html</a></li>
+    <li><a>Css</a></li>
+    <li><a>Javascript</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Mobile Responsive**
+- **Friendly UX**
+- **Easy to use**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here is the Link to this Project's Demo in Loom Video:
+
+> https://www.loom.com/share/ab53647ef0ef41b1915316e15e45fc80
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### 1- Web Browser
+### 2- Code Editor
+### 3- Git
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run This Command:
+
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/Abdulwhbhussain/hello_microverse.git
+```
+
+
+### Install
+
+Install this project with:
+
+Run This Command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+> Open Index.html using Live Server Extension.
+### Run tests
+
+> Coming Soon!
+
+### Deployment
+
+> Here is the Link to this Live Project:
+
+> https://abdulwhbhussain.github.io/Microverse-Capstone-M1/
```

Comment:
- [ ] [OPTIONAL] I highly recommend to include the live demo link under the **_live demo_** section heading ⚒ What do you think about that? 💭

Reply:
Hi @shella12 @KanzaTahreem!
Good to See! **Pakistani** Code Reviewers at **Microverse**;
Both of your **Code Reviews** are **Justified** and Great:

But I messed up My Code By using **Inline Styling** at many places in **HTML Files**;
Because of **Time Limitation**, I am **unable** to Change Everything relevant to Inline styling,
But I will **Try** My Best to **get rid** of this. 

<img src="https://media3.giphy.com/media/29GGY6HStbm81GXc1h/giphy.gif"/>

Reply:
Hello @Abdulwhbhussain Good to see you.
Don't worry about it. You still have some time. I am sure you can fix this 💪 We are here to support you.

Reply:
done.